Address remaining safer CPP warnings in WebCore/platform/mac

cdumez opened the pull request:
#### bd1cd7905b12f7d600c366f8c108945bd7d0d2cf
<pre>
Address remaining safer CPP warnings in WebCore/platform/mac
<a href="https://bugs.webkit.org/show_bug.cgi?id=299271">https://bugs.webkit.org/show_bug.cgi?id=299271</a>

Reviewed by Ryosuke Niwa.

* Source/WebCore/Headers.cmake:
* Source/WebCore/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUncheckedPtrMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUnretainedMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/RetainPtrCtorAdoptCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedLocalVarsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedLambdaCapturesCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedLambdaCapturesCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedLocalVarsCheckerExpectations:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityScrollbar.cpp:
(WebCore::AccessibilityScrollbar::setValue):
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/platform/Cursor.h:
(WebCore::Cursor::ipcData const):
* Source/WebCore/platform/KeyboardScrollingAnimator.cpp:
(WebCore::KeyboardScrollingAnimator::scrollDistance const):
(WebCore::KeyboardScrollingAnimator::scrollingDirections const):
(WebCore::KeyboardScrollingAnimator::beginKeyboardScrollGesture):
(WebCore::KeyboardScrollingAnimator::handleKeyUpEvent):
(WebCore::KeyboardScrollingAnimator::stopScrollingImmediately):
(WebCore::KeyboardScrollingAnimator::checkedScrollableArea const):
* Source/WebCore/platform/KeyboardScrollingAnimator.h:
* Source/WebCore/platform/ScrollAnimator.cpp:
(WebCore::ScrollAnimator::singleAxisScroll):
(WebCore::ScrollAnimator::scrollToPositionWithoutAnimation):
(WebCore::ScrollAnimator::scrollToPositionWithAnimation):
(WebCore::ScrollAnimator::offsetFromPosition const):
(WebCore::ScrollAnimator::positionFromOffset const):
(WebCore::ScrollAnimator::handleWheelEvent):
(WebCore::ScrollAnimator::handleSteppedScrolling):
(WebCore::ScrollAnimator::setCurrentPosition):
(WebCore::ScrollAnimator::notifyPositionChanged):
(WebCore::ScrollAnimator::scrollOffset const):
(WebCore::ScrollAnimator::allowsHorizontalScrolling const):
(WebCore::ScrollAnimator::allowsVerticalScrolling const):
(WebCore::ScrollAnimator::willStartAnimatedScroll):
(WebCore::ScrollAnimator::didStopAnimatedScroll):
(WebCore::ScrollAnimator::stretchAmount const):
(WebCore::ScrollAnimator::edgePinnedState const):
(WebCore::ScrollAnimator::isPinnedOnSide const):
(WebCore::ScrollAnimator::adjustScrollPositionToBoundsIfNecessary):
(WebCore::ScrollAnimator::adjustScrollPositionIfNecessary const):
(WebCore::ScrollAnimator::immediateScrollBy):
(WebCore::ScrollAnimator::scrollExtents const):
(WebCore::ScrollAnimator::pageScaleFactor const):
(WebCore::ScrollAnimator::createTimer):
(WebCore::ScrollAnimator::startAnimationCallback):
(WebCore::ScrollAnimator::scrollAnimationEnabled const):
(WebCore::ScrollAnimator::cancelAnimations):
(WebCore::ScrollAnimator::scrollOffsetAdjustedForSnapping const):
(WebCore::ScrollAnimator::scrollingNodeIDForTesting const):
* Source/WebCore/platform/ScrollAnimator.h:
(WebCore::ScrollAnimator::scrollableArea const):
(WebCore::ScrollAnimator::checkedScrollableArea const):
* Source/WebCore/platform/ScrollView.h:
* Source/WebCore/platform/ScrollableArea.h:
(WebCore::ScrollableArea::protectedHorizontalScrollbar const):
(WebCore::ScrollableArea::protectedVerticalScrollbar const):
* Source/WebCore/platform/Scrollbar.cpp:
(WebCore::Scrollbar::Scrollbar):
(WebCore::Scrollbar::offsetDidChange):
(WebCore::Scrollbar::setFrameRect):
(WebCore::Scrollbar::autoscrollPressedPart):
(WebCore::Scrollbar::moveThumb):
(WebCore::Scrollbar::mouseMoved):
(WebCore::Scrollbar::mouseEntered):
(WebCore::Scrollbar::mouseExited):
(WebCore::Scrollbar::mouseUp):
(WebCore::Scrollbar::mouseDown):
(WebCore::Scrollbar::setEnabled):
(WebCore::Scrollbar::shouldParticipateInHitTesting):
(WebCore::Scrollbar::isWindowActive const):
(WebCore::Scrollbar::invalidateRect):
(WebCore::Scrollbar::convertToContainingView const):
(WebCore::Scrollbar::convertFromContainingView const):
(WebCore::Scrollbar::supportsUpdateOnSecondaryThread const):
(WebCore::Scrollbar::deviceScaleFactor const):
(WebCore::Scrollbar::shouldRegisterScrollbar const):
(WebCore::Scrollbar::minimumThumbLength const):
(WebCore::Scrollbar::updateScrollbarThickness):
* Source/WebCore/platform/Scrollbar.h:
(WebCore::Scrollbar::scrollableArea const): Deleted.
* Source/WebCore/platform/ScrollbarInlines.h: Copied from Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h.
(WebCore::Scrollbar::scrollableArea const):
(WebCore::Scrollbar::checkedScrollableArea const):
* Source/WebCore/platform/ScrollbarsController.cpp:
(WebCore::ScrollbarsController::shouldSuspendScrollbarAnimations const):
(WebCore::ScrollbarsController::updateScrollbarsThickness):
* Source/WebCore/platform/ScrollbarsController.h:
(WebCore::ScrollbarsController::scrollableArea const): Deleted.
* Source/WebCore/platform/ScrollbarsControllerInlines.h: Copied from Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h.
(WebCore::ScrollbarsController::scrollableArea const):
(WebCore::ScrollbarsController::checkedScrollableArea const):
* Source/WebCore/platform/Widget.h:
* Source/WebCore/platform/adwaita/ScrollbarThemeAdwaita.cpp:
* Source/WebCore/platform/cocoa/VideoPresentationLayerProvider.h:
(WebCore::VideoPresentationLayerProvider::protectedLayerHostView const):
* Source/WebCore/platform/graphics/GraphicsLayer.cpp:
(WebCore::GraphicsLayer::protectedPlatformLayer const):
* Source/WebCore/platform/graphics/GraphicsLayer.h:
* Source/WebCore/platform/gtk/ScrollbarThemeGtk.cpp:
* Source/WebCore/platform/ios/ScrollAnimatorIOS.mm:
(WebCore::ScrollAnimatorIOS::handleTouchEvent):
(WebCore::ScrollAnimatorIOS::determineScrollableAreaForTouchSequence):
* Source/WebCore/platform/ios/ScrollViewIOS.mm:
(WebCore::ScrollView::protectedScrollView const):
(WebCore::ScrollView::protectedDocumentView const):
* Source/WebCore/platform/ios/WidgetIOS.mm:
(WebCore::Widget::show):
(WebCore::Widget::hide):
(WebCore::Widget::frameRect const):
(WebCore::Widget::setFrameRect):
(WebCore::Widget::outerView const):
(WebCore::Widget::paint):
(WebCore::Widget::addToSuperview):
(WebCore::Widget::removeFromSuperview):
(WebCore::Widget::protectedPlatformWidget const):
(WebCore::Widget::getOuterView const): Deleted.
* Source/WebCore/platform/mac/CursorMac.mm:
(WebCore::createCoreCursorClassSingleton):
(WebCore::coreCursorClassSingleton):
(WebCore::cursor):
(WebCore::Cursor::setAsPlatformCursor const):
* Source/WebCore/platform/mac/DataDetectorHighlight.mm:
(WebCore::DataDetectorHighlight::paintContents):
* Source/WebCore/platform/mac/HIDDevice.cpp:
(WebCore::getDevicePropertyAsInt):
(WebCore::HIDDevice::HIDDevice):
(WebCore::HIDDevice::uniqueInputElementsInDeviceTreeOrder const):
* Source/WebCore/platform/mac/PasteboardMac.mm:
(WebCore::fileWrapper):
(WebCore::writeFileWrapperAsRTFDAttachment):
(WebCore::Pasteboard::write):
(WebCore::setDragImageImpl):
(WebCore::Pasteboard::setDragImage):
* Source/WebCore/platform/mac/PasteboardWriter.mm:
(WebCore::createPasteboardWriter):
* Source/WebCore/platform/mac/PlatformEventFactoryMac.mm:
(WebCore::globalPoint):
(WebCore::unadjustedMovementForEvent):
* Source/WebCore/platform/mac/PowerObserverMac.cpp:
(WebCore::PowerObserver::didReceiveSystemPowerNotification):
* Source/WebCore/platform/mac/ScrollAnimatorMac.mm:
(WebCore::ScrollAnimatorMac::handleWheelEventPhase):
(WebCore::ScrollAnimatorMac::handleWheelEvent):
(WebCore::ScrollAnimatorMac::allowsVerticalStretching const):
(WebCore::ScrollAnimatorMac::allowsHorizontalStretching const):
* Source/WebCore/platform/mac/ScrollViewMac.mm:
(WebCore::ScrollView::protectedScrollView const):
(WebCore::ScrollView::documentView const):
(WebCore::ScrollView::protectedDocumentView const):
(WebCore::ScrollView::platformAddChild):
(WebCore::ScrollView::platformSetScrollbarModes):
(WebCore::ScrollView::platformScrollbarModes const):
(WebCore::ScrollView::platformSetCanBlitOnScroll):
(WebCore::ScrollView::platformCanBlitOnScroll const):
(WebCore::ScrollView::platformContentInsets const):
(WebCore::ScrollView::platformSetContentInsets):
(WebCore::ScrollView::platformVisibleContentRect const):
(WebCore::ScrollView::platformVisibleContentRectIncludingObscuredArea const):
(WebCore::ScrollView::platformSetContentsSize):
(WebCore::ScrollView::platformSetScrollbarsSuppressed):
(WebCore::ScrollView::platformSetScrollPosition):
(WebCore::ScrollView::platformRepaintContentRectangle):
(WebCore::ScrollView::platformContentsToScreen const):
(WebCore::ScrollView::platformScreenToContents const):
(WebCore::ScrollView::platformIsOffscreen const):
(WebCore::ScrollView::platformSetScrollbarOverlayStyle):
(WebCore::ScrollView::platformSetScrollOrigin):
* Source/WebCore/platform/mac/ScrollbarMac.h:
* Source/WebCore/platform/mac/ScrollbarMac.mm:
(WebCore::ScrollbarMac::protectedScrollerImp const):
* Source/WebCore/platform/mac/ScrollbarThemeMac.h:
* Source/WebCore/platform/mac/ScrollbarThemeMac.mm:
(WebCore::ScrollbarThemeMac::didCreateScrollerImp):
(WebCore::ScrollbarThemeMac::protectedScrollerImpForScrollbar):
(WebCore::ScrollbarThemeMac::isLayoutDirectionRTL):
(WebCore::ScrollbarThemeMac::updateScrollbarOverlayStyle):
(WebCore::ScrollbarThemeMac::hasThumb):
(WebCore::ScrollbarThemeMac::minimumThumbLength):
(WebCore::ScrollbarThemeMac::updateEnabledState):
(WebCore::ScrollbarThemeMac::setPaintCharacteristicsForScrollbar):
(WebCore::paintScrollbar):
(WebCore::ScrollbarThemeMac::setUpOverhangAreaShadow):
* Source/WebCore/platform/mac/ScrollbarsControllerMac.mm:
(-[WebScrollerImpPairDelegate scrollerImpPair:updateScrollerStyleForNewRecommendedScrollerStyle:]):
(-[WebScrollbarPartAnimation initWithScrollbar:featureToAnimate:animateFrom:animateTo:duration:]):
(-[WebScrollerImpDelegate scrollbarsController]):
(-[WebScrollerImpDelegate layer]):
(-[WebScrollerImpDelegate mouseLocationInScrollerForScrollerImp:]):
(-[WebScrollerImpDelegate shouldUseLayerPerPartForScrollerImp:]):
(-[WebScrollerImpDelegate effectiveAppearanceForScrollerImp:]):
(-[WebScrollerImpDelegate setUpAlphaAnimation:scrollerPainter:part:animateAlphaTo:duration:]):
(-[WebScrollerImpDelegate scrollerImp:animateKnobAlphaTo:duration:]):
(-[WebScrollerImpDelegate scrollerImp:animateUIStateTransitionWithDuration:]):
(-[WebScrollerImpDelegate scrollerImp:animateExpansionTransitionWithDuration:]):
(-[WebScrollerImpDelegate invalidate]):
(WebCore::ScrollbarsControllerMac::setVisibleScrollerThumbRect):
(WebCore::ScrollbarsControllerMac::mouseEnteredScrollbar const):
(WebCore::ScrollbarsControllerMac::mouseExitedScrollbar const):
(WebCore::ScrollbarsControllerMac::mouseIsDownInScrollbar const):
(WebCore::ScrollbarsControllerMac::didBeginScrollGesture):
(WebCore::ScrollbarsControllerMac::didEndScrollGesture):
(WebCore::ScrollbarsControllerMac::didAddVerticalScrollbar):
(WebCore::ScrollbarsControllerMac::willRemoveVerticalScrollbar):
(WebCore::ScrollbarsControllerMac::didAddHorizontalScrollbar):
(WebCore::ScrollbarsControllerMac::willRemoveHorizontalScrollbar):
(WebCore::ScrollbarsControllerMac::invalidateScrollbarPartLayers):
(WebCore::ScrollbarsControllerMac::verticalScrollbarLayerDidChange):
(WebCore::ScrollbarsControllerMac::horizontalScrollbarLayerDidChange):
(WebCore::ScrollbarsControllerMac::shouldScrollbarParticipateInHitTesting):
(WebCore::ScrollbarsControllerMac::notifyContentAreaScrolled):
(WebCore::ScrollbarsControllerMac::updateScrollerImps):
(WebCore::ScrollbarsControllerMac::updateScrollerStyle):
(WebCore::ScrollbarsControllerMac::sendContentAreaScrolledTimerFired):
(WebCore::ScrollbarsControllerMac::sendContentAreaScrolledSoon):
(WebCore::scrollbarState):
(WebCore::ScrollbarsControllerMac::horizontalScrollbarStateForTesting const):
(WebCore::ScrollbarsControllerMac::verticalScrollbarStateForTesting const):
(WebCore::ScrollbarsControllerMac::wheelEventTestMonitor const):
* Source/WebCore/platform/mac/ScrollingEffectsController.mm:
(WebCore::ScrollingEffectsController::handleWheelEvent):
* Source/WebCore/platform/mac/ScrollingMomentumCalculatorMac.h:
* Source/WebCore/platform/mac/ScrollingMomentumCalculatorMac.mm:
(WebCore::ScrollingMomentumCalculatorMac::scrollOffsetAfterElapsedTime):
(WebCore::ScrollingMomentumCalculatorMac::setMomentumCalculatorDestinationOffset):
(WebCore::ScrollingMomentumCalculatorMac::animationDuration):
(WebCore::ScrollingMomentumCalculatorMac::ensureProtectedPlatformMomentumCalculator):
* Source/WebCore/platform/mac/SerializedPlatformDataCueMac.h:
(isType):
* Source/WebCore/platform/mac/SerializedPlatformDataCueMac.mm:
(WebCore::SerializedPlatformDataCueMac::deserialize const):
(WebCore::SerializedPlatformDataCueMac::isEqual const):
(WebCore::jsValueWithArrayInContext):
(WebCore::jsValueWithDictionaryInContext):
(WebCore::toSerializedPlatformDataCueMac): Deleted.
* Source/WebCore/platform/mac/VideoPresentationInterfaceMac.h:
* Source/WebCore/platform/mac/VideoPresentationInterfaceMac.mm:
(-[WebVideoViewContainer resizeWithOldSuperviewSize:]):
(-[WebVideoViewContainer viewDidMoveToSuperview]):
(-[WebVideoPresentationInterfaceMacObjC setUpPIPForVideoView:withFrame:inWindow:]):
(-[WebVideoPresentationInterfaceMacObjC enterPIP]):
(-[WebVideoPresentationInterfaceMacObjC exitPIP]):
(-[WebVideoPresentationInterfaceMacObjC boundsDidChangeForVideoViewContainer:]):
(-[WebVideoPresentationInterfaceMacObjC pipShouldClose:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionSkipPreroll:]):
(-[WebVideoPresentationInterfaceMacObjC pipDidClose:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionPlay:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionPause:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionStop:]):
(-[WebSeekableVideoPresentationInterfaceMacObjC pipAction:skipInterval:]):
(WebCore::VideoPresentationInterfaceMac::VideoPresentationInterfaceMac):
(WebCore::VideoPresentationInterfaceMac::~VideoPresentationInterfaceMac):
(WebCore::VideoPresentationInterfaceMac::durationChanged):
(WebCore::VideoPresentationInterfaceMac::currentTimeChanged):
(WebCore::VideoPresentationInterfaceMac::rateChanged):
(WebCore::VideoPresentationInterfaceMac::videoPresentationInterfaceObjC):
(WebCore::VideoPresentationInterfaceMac::protectedVideoPresentationInterfaceObjC):
(WebCore::VideoPresentationInterfaceMac::setupFullscreen):
(WebCore::VideoPresentationInterfaceMac::enterFullscreen):
(WebCore::VideoPresentationInterfaceMac::exitFullscreen):
(WebCore::VideoPresentationInterfaceMac::exitFullscreenWithoutAnimationToMode):
(WebCore::VideoPresentationInterfaceMac::requestHideAndExitPiP):
* Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h:
* Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.mm:
(-[WebCoreFullScreenPlaceholderView mouseDown:]):
* Source/WebCore/platform/mac/WebCoreFullScreenWarningView.mm:
* Source/WebCore/platform/mac/WebCoreFullScreenWindow.mm:
(-[WebCoreFullScreenWindow setStyleMask:]):
* Source/WebCore/platform/mac/WebCoreNSFontManagerExtras.mm:
(WebCore::computedFontChanges):
(WebCore::computedFontAttributeChanges):
* Source/WebCore/platform/mac/WebCoreObjCExtras.mm:
(WebCoreObjCScheduleDeallocateOnMainThread):
(WebCoreObjCScheduleDeallocateOnMainRunLoop):
* Source/WebCore/platform/mac/WebPlaybackControlsManager.h:
* Source/WebCore/platform/mac/WebPlaybackControlsManager.mm:
(-[WebPlaybackControlsManager dealloc]):
(-[WebPlaybackControlsManager seekToTime:toleranceBefore:toleranceAfter:]):
(-[WebPlaybackControlsManager beginTouchBarScrubbing]):
(-[WebPlaybackControlsManager endTouchBarScrubbing]):
(-[WebPlaybackControlsManager setCurrentAudioTouchBarMediaSelectionOption:]):
(-[WebPlaybackControlsManager setCurrentLegibleTouchBarMediaSelectionOption:]):
(-[WebPlaybackControlsManager setPlaybackSessionInterfaceMac:]):
(-[WebPlaybackControlsManager togglePlayback]):
(-[WebPlaybackControlsManager setPlaying:]):
(-[WebPlaybackControlsManager setDefaultPlaybackRate:fromJavaScript:]):
(-[WebPlaybackControlsManager setRate:fromJavaScript:]):
(-[WebPlaybackControlsManager togglePictureInPicture]):
(-[WebPlaybackControlsManager enterInWindow]):
(-[WebPlaybackControlsManager exitInWindow]):
* Source/WebCore/platform/mac/WidgetMac.mm:
(WebCore::Widget::setFocus):
(WebCore::Widget::show):
(WebCore::Widget::hide):
(WebCore::Widget::frameRect const):
(WebCore::Widget::setFrameRect):
(WebCore::Widget::outerView const):
(WebCore::Widget::protectedOuterView const):
(WebCore::Widget::paint):
(WebCore::Widget::setIsSelected):
(WebCore::Widget::removeFromSuperview):
(WebCore::Widget::convertFromRootToContainingWindow):
(WebCore::Widget::convertFromContainingWindowToRoot):
(WebCore::Widget::protectedPlatformWidget const):
(WebCore::Widget::getOuterView const): Deleted.
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebKit/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm:
(WebKit::RemoteScrollbarsController::scrollbarLayoutDirectionChanged):
(WebKit::RemoteScrollbarsController::mouseEnteredContentArea):
(WebKit::RemoteScrollbarsController::mouseExitedContentArea):
(WebKit::RemoteScrollbarsController::mouseMovedInContentArea):
(WebKit::RemoteScrollbarsController::shouldRegisterScrollbars const):
(WebKit::RemoteScrollbarsController::scrollbarWidthChanged):
(WebKit::RemoteScrollbarsController::updateScrollbarStyle):
(WebKit::RemoteScrollbarsController::scrollbarColorChanged):

Canonical link: <a href="https://commits.webkit.org/300377@main">https://commits.webkit.org/300377@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/4347dc78ac7f5f00fe91608503cdf486bce35695

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [❌ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/122337 "15 style errors") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/42041 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/32729 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/128923 "Built successfully") | [  ~~🛠 win~~](https://ews-build.webkit.org/#/builders/59/builds/74436 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/f0d82d10-fbe0-4bde-98f5-67590b4d6381) 
| [✅ 🧪 bindings](https://ews-build.webkit.org/#/builders/9/builds/124213 "Passed tests") | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/42762 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/50636 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/92990 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/59/builds/74436 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/16a1de0c-116f-4ab8-b144-e1d229af7344) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/125289 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/34107 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/109544 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/73648 "Passed tests") | | [✅ 🛠 vision-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/1cbfa4c1-78ba-427a-8cb3-eb55252b277c) 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/33100 "Passed tests") | [✅ 🧪 mac-wk1](https://ews-build.webkit.org/#/builders/135/builds/27703 "Passed tests") | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/72414 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/103683 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/27911 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/131664 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/49278 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/37499 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/101540 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/49652 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/105764 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/101411 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/25722 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/46783 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/24914 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/46039 "Build was cancelled. Recent messages:Printed configuration") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/49135 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/54878 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/48603 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/51953 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/50284 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/c285ddd0f898d9ab40599eccc0d165e99dbe46e2. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51098

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/28651f242a7a9e4fb57bea79974690c40694d704. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51098

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ff4825049d4ca1ef4380f219a0d26209b04ef037. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/51098

cdumez committed:
Address remaining safer CPP warnings in WebCore/platform/mac
https://bugs.webkit.org/show_bug.cgi?id=299271

Reviewed by Ryosuke Niwa.

* Source/WebCore/Headers.cmake:
* Source/WebCore/SaferCPPExpectations/MemoryUnsafeCastCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUncheckedPtrMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUncountedMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/NoUnretainedMemberCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/RetainPtrCtorAdoptCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncheckedLocalVarsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UncountedLambdaCapturesCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedCallArgsCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedLambdaCapturesCheckerExpectations:
* Source/WebCore/SaferCPPExpectations/UnretainedLocalVarsCheckerExpectations:
* Source/WebCore/WebCore.xcodeproj/project.pbxproj:
* Source/WebCore/accessibility/AccessibilityScrollbar.cpp:
(WebCore::AccessibilityScrollbar::setValue):
* Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp:
* Source/WebCore/platform/Cursor.h:
(WebCore::Cursor::ipcData const):
* Source/WebCore/platform/KeyboardScrollingAnimator.cpp:
(WebCore::KeyboardScrollingAnimator::scrollDistance const):
(WebCore::KeyboardScrollingAnimator::scrollingDirections const):
(WebCore::KeyboardScrollingAnimator::beginKeyboardScrollGesture):
(WebCore::KeyboardScrollingAnimator::handleKeyUpEvent):
(WebCore::KeyboardScrollingAnimator::stopScrollingImmediately):
(WebCore::KeyboardScrollingAnimator::checkedScrollableArea const):
* Source/WebCore/platform/KeyboardScrollingAnimator.h:
* Source/WebCore/platform/ScrollAnimator.cpp:
(WebCore::ScrollAnimator::singleAxisScroll):
(WebCore::ScrollAnimator::scrollToPositionWithoutAnimation):
(WebCore::ScrollAnimator::scrollToPositionWithAnimation):
(WebCore::ScrollAnimator::offsetFromPosition const):
(WebCore::ScrollAnimator::positionFromOffset const):
(WebCore::ScrollAnimator::handleWheelEvent):
(WebCore::ScrollAnimator::handleSteppedScrolling):
(WebCore::ScrollAnimator::setCurrentPosition):
(WebCore::ScrollAnimator::notifyPositionChanged):
(WebCore::ScrollAnimator::scrollOffset const):
(WebCore::ScrollAnimator::allowsHorizontalScrolling const):
(WebCore::ScrollAnimator::allowsVerticalScrolling const):
(WebCore::ScrollAnimator::willStartAnimatedScroll):
(WebCore::ScrollAnimator::didStopAnimatedScroll):
(WebCore::ScrollAnimator::stretchAmount const):
(WebCore::ScrollAnimator::edgePinnedState const):
(WebCore::ScrollAnimator::isPinnedOnSide const):
(WebCore::ScrollAnimator::adjustScrollPositionToBoundsIfNecessary):
(WebCore::ScrollAnimator::adjustScrollPositionIfNecessary const):
(WebCore::ScrollAnimator::immediateScrollBy):
(WebCore::ScrollAnimator::scrollExtents const):
(WebCore::ScrollAnimator::pageScaleFactor const):
(WebCore::ScrollAnimator::createTimer):
(WebCore::ScrollAnimator::startAnimationCallback):
(WebCore::ScrollAnimator::scrollAnimationEnabled const):
(WebCore::ScrollAnimator::cancelAnimations):
(WebCore::ScrollAnimator::scrollOffsetAdjustedForSnapping const):
(WebCore::ScrollAnimator::scrollingNodeIDForTesting const):
* Source/WebCore/platform/ScrollAnimator.h:
(WebCore::ScrollAnimator::scrollableArea const):
(WebCore::ScrollAnimator::checkedScrollableArea const):
* Source/WebCore/platform/ScrollView.h:
* Source/WebCore/platform/ScrollableArea.h:
(WebCore::ScrollableArea::protectedHorizontalScrollbar const):
(WebCore::ScrollableArea::protectedVerticalScrollbar const):
* Source/WebCore/platform/Scrollbar.cpp:
(WebCore::Scrollbar::Scrollbar):
(WebCore::Scrollbar::offsetDidChange):
(WebCore::Scrollbar::setFrameRect):
(WebCore::Scrollbar::autoscrollPressedPart):
(WebCore::Scrollbar::moveThumb):
(WebCore::Scrollbar::mouseMoved):
(WebCore::Scrollbar::mouseEntered):
(WebCore::Scrollbar::mouseExited):
(WebCore::Scrollbar::mouseUp):
(WebCore::Scrollbar::mouseDown):
(WebCore::Scrollbar::setEnabled):
(WebCore::Scrollbar::shouldParticipateInHitTesting):
(WebCore::Scrollbar::isWindowActive const):
(WebCore::Scrollbar::invalidateRect):
(WebCore::Scrollbar::convertToContainingView const):
(WebCore::Scrollbar::convertFromContainingView const):
(WebCore::Scrollbar::supportsUpdateOnSecondaryThread const):
(WebCore::Scrollbar::deviceScaleFactor const):
(WebCore::Scrollbar::shouldRegisterScrollbar const):
(WebCore::Scrollbar::minimumThumbLength const):
(WebCore::Scrollbar::updateScrollbarThickness):
* Source/WebCore/platform/Scrollbar.h:
(WebCore::Scrollbar::scrollableArea const): Deleted.
* Source/WebCore/platform/ScrollbarInlines.h: Copied from Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h.
(WebCore::Scrollbar::scrollableArea const):
(WebCore::Scrollbar::checkedScrollableArea const):
* Source/WebCore/platform/ScrollbarsController.cpp:
(WebCore::ScrollbarsController::shouldSuspendScrollbarAnimations const):
(WebCore::ScrollbarsController::updateScrollbarsThickness):
* Source/WebCore/platform/ScrollbarsController.h:
(WebCore::ScrollbarsController::scrollableArea const): Deleted.
* Source/WebCore/platform/ScrollbarsControllerInlines.h: Copied from Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h.
(WebCore::ScrollbarsController::scrollableArea const):
(WebCore::ScrollbarsController::checkedScrollableArea const):
* Source/WebCore/platform/Widget.h:
* Source/WebCore/platform/adwaita/ScrollbarThemeAdwaita.cpp:
* Source/WebCore/platform/cocoa/VideoPresentationLayerProvider.h:
(WebCore::VideoPresentationLayerProvider::protectedLayerHostView const):
* Source/WebCore/platform/graphics/GraphicsLayer.cpp:
(WebCore::GraphicsLayer::protectedPlatformLayer const):
* Source/WebCore/platform/graphics/GraphicsLayer.h:
* Source/WebCore/platform/gtk/ScrollbarThemeGtk.cpp:
* Source/WebCore/platform/ios/ScrollAnimatorIOS.mm:
(WebCore::ScrollAnimatorIOS::handleTouchEvent):
(WebCore::ScrollAnimatorIOS::determineScrollableAreaForTouchSequence):
* Source/WebCore/platform/ios/ScrollViewIOS.mm:
(WebCore::ScrollView::protectedScrollView const):
(WebCore::ScrollView::protectedDocumentView const):
* Source/WebCore/platform/ios/WidgetIOS.mm:
(WebCore::Widget::show):
(WebCore::Widget::hide):
(WebCore::Widget::frameRect const):
(WebCore::Widget::setFrameRect):
(WebCore::Widget::outerView const):
(WebCore::Widget::paint):
(WebCore::Widget::addToSuperview):
(WebCore::Widget::removeFromSuperview):
(WebCore::Widget::protectedPlatformWidget const):
(WebCore::Widget::getOuterView const): Deleted.
* Source/WebCore/platform/mac/CursorMac.mm:
(WebCore::createCoreCursorClassSingleton):
(WebCore::coreCursorClassSingleton):
(WebCore::cursor):
(WebCore::Cursor::setAsPlatformCursor const):
* Source/WebCore/platform/mac/DataDetectorHighlight.mm:
(WebCore::DataDetectorHighlight::paintContents):
* Source/WebCore/platform/mac/HIDDevice.cpp:
(WebCore::getDevicePropertyAsInt):
(WebCore::HIDDevice::HIDDevice):
(WebCore::HIDDevice::uniqueInputElementsInDeviceTreeOrder const):
* Source/WebCore/platform/mac/PasteboardMac.mm:
(WebCore::fileWrapper):
(WebCore::writeFileWrapperAsRTFDAttachment):
(WebCore::Pasteboard::write):
(WebCore::setDragImageImpl):
(WebCore::Pasteboard::setDragImage):
* Source/WebCore/platform/mac/PasteboardWriter.mm:
(WebCore::createPasteboardWriter):
* Source/WebCore/platform/mac/PlatformEventFactoryMac.mm:
(WebCore::globalPoint):
(WebCore::unadjustedMovementForEvent):
* Source/WebCore/platform/mac/PowerObserverMac.cpp:
(WebCore::PowerObserver::didReceiveSystemPowerNotification):
* Source/WebCore/platform/mac/ScrollAnimatorMac.mm:
(WebCore::ScrollAnimatorMac::handleWheelEventPhase):
(WebCore::ScrollAnimatorMac::handleWheelEvent):
(WebCore::ScrollAnimatorMac::allowsVerticalStretching const):
(WebCore::ScrollAnimatorMac::allowsHorizontalStretching const):
* Source/WebCore/platform/mac/ScrollViewMac.mm:
(WebCore::ScrollView::protectedScrollView const):
(WebCore::ScrollView::documentView const):
(WebCore::ScrollView::protectedDocumentView const):
(WebCore::ScrollView::platformAddChild):
(WebCore::ScrollView::platformSetScrollbarModes):
(WebCore::ScrollView::platformScrollbarModes const):
(WebCore::ScrollView::platformSetCanBlitOnScroll):
(WebCore::ScrollView::platformCanBlitOnScroll const):
(WebCore::ScrollView::platformContentInsets const):
(WebCore::ScrollView::platformSetContentInsets):
(WebCore::ScrollView::platformVisibleContentRect const):
(WebCore::ScrollView::platformVisibleContentRectIncludingObscuredArea const):
(WebCore::ScrollView::platformSetContentsSize):
(WebCore::ScrollView::platformSetScrollbarsSuppressed):
(WebCore::ScrollView::platformSetScrollPosition):
(WebCore::ScrollView::platformRepaintContentRectangle):
(WebCore::ScrollView::platformContentsToScreen const):
(WebCore::ScrollView::platformScreenToContents const):
(WebCore::ScrollView::platformIsOffscreen const):
(WebCore::ScrollView::platformSetScrollbarOverlayStyle):
(WebCore::ScrollView::platformSetScrollOrigin):
* Source/WebCore/platform/mac/ScrollbarMac.h:
* Source/WebCore/platform/mac/ScrollbarMac.mm:
(WebCore::ScrollbarMac::protectedScrollerImp const):
* Source/WebCore/platform/mac/ScrollbarThemeMac.h:
* Source/WebCore/platform/mac/ScrollbarThemeMac.mm:
(WebCore::ScrollbarThemeMac::didCreateScrollerImp):
(WebCore::ScrollbarThemeMac::protectedScrollerImpForScrollbar):
(WebCore::ScrollbarThemeMac::isLayoutDirectionRTL):
(WebCore::ScrollbarThemeMac::updateScrollbarOverlayStyle):
(WebCore::ScrollbarThemeMac::hasThumb):
(WebCore::ScrollbarThemeMac::minimumThumbLength):
(WebCore::ScrollbarThemeMac::updateEnabledState):
(WebCore::ScrollbarThemeMac::setPaintCharacteristicsForScrollbar):
(WebCore::paintScrollbar):
(WebCore::ScrollbarThemeMac::setUpOverhangAreaShadow):
* Source/WebCore/platform/mac/ScrollbarsControllerMac.mm:
(-[WebScrollerImpPairDelegate scrollerImpPair:updateScrollerStyleForNewRecommendedScrollerStyle:]):
(-[WebScrollbarPartAnimation initWithScrollbar:featureToAnimate:animateFrom:animateTo:duration:]):
(-[WebScrollerImpDelegate scrollbarsController]):
(-[WebScrollerImpDelegate layer]):
(-[WebScrollerImpDelegate mouseLocationInScrollerForScrollerImp:]):
(-[WebScrollerImpDelegate shouldUseLayerPerPartForScrollerImp:]):
(-[WebScrollerImpDelegate effectiveAppearanceForScrollerImp:]):
(-[WebScrollerImpDelegate setUpAlphaAnimation:scrollerPainter:part:animateAlphaTo:duration:]):
(-[WebScrollerImpDelegate scrollerImp:animateKnobAlphaTo:duration:]):
(-[WebScrollerImpDelegate scrollerImp:animateUIStateTransitionWithDuration:]):
(-[WebScrollerImpDelegate scrollerImp:animateExpansionTransitionWithDuration:]):
(-[WebScrollerImpDelegate invalidate]):
(WebCore::ScrollbarsControllerMac::setVisibleScrollerThumbRect):
(WebCore::ScrollbarsControllerMac::mouseEnteredScrollbar const):
(WebCore::ScrollbarsControllerMac::mouseExitedScrollbar const):
(WebCore::ScrollbarsControllerMac::mouseIsDownInScrollbar const):
(WebCore::ScrollbarsControllerMac::didBeginScrollGesture):
(WebCore::ScrollbarsControllerMac::didEndScrollGesture):
(WebCore::ScrollbarsControllerMac::didAddVerticalScrollbar):
(WebCore::ScrollbarsControllerMac::willRemoveVerticalScrollbar):
(WebCore::ScrollbarsControllerMac::didAddHorizontalScrollbar):
(WebCore::ScrollbarsControllerMac::willRemoveHorizontalScrollbar):
(WebCore::ScrollbarsControllerMac::invalidateScrollbarPartLayers):
(WebCore::ScrollbarsControllerMac::verticalScrollbarLayerDidChange):
(WebCore::ScrollbarsControllerMac::horizontalScrollbarLayerDidChange):
(WebCore::ScrollbarsControllerMac::shouldScrollbarParticipateInHitTesting):
(WebCore::ScrollbarsControllerMac::notifyContentAreaScrolled):
(WebCore::ScrollbarsControllerMac::updateScrollerImps):
(WebCore::ScrollbarsControllerMac::updateScrollerStyle):
(WebCore::ScrollbarsControllerMac::sendContentAreaScrolledTimerFired):
(WebCore::ScrollbarsControllerMac::sendContentAreaScrolledSoon):
(WebCore::scrollbarState):
(WebCore::ScrollbarsControllerMac::horizontalScrollbarStateForTesting const):
(WebCore::ScrollbarsControllerMac::verticalScrollbarStateForTesting const):
(WebCore::ScrollbarsControllerMac::wheelEventTestMonitor const):
* Source/WebCore/platform/mac/ScrollingEffectsController.mm:
(WebCore::ScrollingEffectsController::handleWheelEvent):
* Source/WebCore/platform/mac/ScrollingMomentumCalculatorMac.h:
* Source/WebCore/platform/mac/ScrollingMomentumCalculatorMac.mm:
(WebCore::ScrollingMomentumCalculatorMac::scrollOffsetAfterElapsedTime):
(WebCore::ScrollingMomentumCalculatorMac::setMomentumCalculatorDestinationOffset):
(WebCore::ScrollingMomentumCalculatorMac::animationDuration):
(WebCore::ScrollingMomentumCalculatorMac::ensureProtectedPlatformMomentumCalculator):
* Source/WebCore/platform/mac/SerializedPlatformDataCueMac.h:
(isType):
* Source/WebCore/platform/mac/SerializedPlatformDataCueMac.mm:
(WebCore::SerializedPlatformDataCueMac::deserialize const):
(WebCore::SerializedPlatformDataCueMac::isEqual const):
(WebCore::jsValueWithArrayInContext):
(WebCore::jsValueWithDictionaryInContext):
(WebCore::toSerializedPlatformDataCueMac): Deleted.
* Source/WebCore/platform/mac/VideoPresentationInterfaceMac.h:
* Source/WebCore/platform/mac/VideoPresentationInterfaceMac.mm:
(-[WebVideoViewContainer resizeWithOldSuperviewSize:]):
(-[WebVideoViewContainer viewDidMoveToSuperview]):
(-[WebVideoPresentationInterfaceMacObjC setUpPIPForVideoView:withFrame:inWindow:]):
(-[WebVideoPresentationInterfaceMacObjC enterPIP]):
(-[WebVideoPresentationInterfaceMacObjC exitPIP]):
(-[WebVideoPresentationInterfaceMacObjC boundsDidChangeForVideoViewContainer:]):
(-[WebVideoPresentationInterfaceMacObjC pipShouldClose:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionSkipPreroll:]):
(-[WebVideoPresentationInterfaceMacObjC pipDidClose:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionPlay:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionPause:]):
(-[WebVideoPresentationInterfaceMacObjC pipActionStop:]):
(-[WebSeekableVideoPresentationInterfaceMacObjC pipAction:skipInterval:]):
(WebCore::VideoPresentationInterfaceMac::VideoPresentationInterfaceMac):
(WebCore::VideoPresentationInterfaceMac::~VideoPresentationInterfaceMac):
(WebCore::VideoPresentationInterfaceMac::durationChanged):
(WebCore::VideoPresentationInterfaceMac::currentTimeChanged):
(WebCore::VideoPresentationInterfaceMac::rateChanged):
(WebCore::VideoPresentationInterfaceMac::videoPresentationInterfaceObjC):
(WebCore::VideoPresentationInterfaceMac::protectedVideoPresentationInterfaceObjC):
(WebCore::VideoPresentationInterfaceMac::setupFullscreen):
(WebCore::VideoPresentationInterfaceMac::enterFullscreen):
(WebCore::VideoPresentationInterfaceMac::exitFullscreen):
(WebCore::VideoPresentationInterfaceMac::exitFullscreenWithoutAnimationToMode):
(WebCore::VideoPresentationInterfaceMac::requestHideAndExitPiP):
* Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.h:
* Source/WebCore/platform/mac/WebCoreFullScreenPlaceholderView.mm:
(-[WebCoreFullScreenPlaceholderView mouseDown:]):
* Source/WebCore/platform/mac/WebCoreFullScreenWarningView.mm:
* Source/WebCore/platform/mac/WebCoreFullScreenWindow.mm:
(-[WebCoreFullScreenWindow setStyleMask:]):
* Source/WebCore/platform/mac/WebCoreNSFontManagerExtras.mm:
(WebCore::computedFontChanges):
(WebCore::computedFontAttributeChanges):
* Source/WebCore/platform/mac/WebCoreObjCExtras.mm:
(WebCoreObjCScheduleDeallocateOnMainThread):
(WebCoreObjCScheduleDeallocateOnMainRunLoop):
* Source/WebCore/platform/mac/WebPlaybackControlsManager.h:
* Source/WebCore/platform/mac/WebPlaybackControlsManager.mm:
(-[WebPlaybackControlsManager dealloc]):
(-[WebPlaybackControlsManager seekToTime:toleranceBefore:toleranceAfter:]):
(-[WebPlaybackControlsManager beginTouchBarScrubbing]):
(-[WebPlaybackControlsManager endTouchBarScrubbing]):
(-[WebPlaybackControlsManager setCurrentAudioTouchBarMediaSelectionOption:]):
(-[WebPlaybackControlsManager setCurrentLegibleTouchBarMediaSelectionOption:]):
(-[WebPlaybackControlsManager setPlaybackSessionInterfaceMac:]):
(-[WebPlaybackControlsManager togglePlayback]):
(-[WebPlaybackControlsManager setPlaying:]):
(-[WebPlaybackControlsManager setDefaultPlaybackRate:fromJavaScript:]):
(-[WebPlaybackControlsManager setRate:fromJavaScript:]):
(-[WebPlaybackControlsManager togglePictureInPicture]):
(-[WebPlaybackControlsManager enterInWindow]):
(-[WebPlaybackControlsManager exitInWindow]):
* Source/WebCore/platform/mac/WidgetMac.mm:
(WebCore::Widget::setFocus):
(WebCore::Widget::show):
(WebCore::Widget::hide):
(WebCore::Widget::frameRect const):
(WebCore::Widget::setFrameRect):
(WebCore::Widget::outerView const):
(WebCore::Widget::protectedOuterView const):
(WebCore::Widget::paint):
(WebCore::Widget::setIsSelected):
(WebCore::Widget::removeFromSuperview):
(WebCore::Widget::convertFromRootToContainingWindow):
(WebCore::Widget::convertFromContainingWindowToRoot):
(WebCore::Widget::protectedPlatformWidget const):
(WebCore::Widget::getOuterView const): Deleted.
* Source/WebCore/rendering/RenderScrollbar.cpp:
* Source/WebKit/SaferCPPExpectations/UncheckedCallArgsCheckerExpectations:
* Source/WebKit/WebProcess/WebPage/RemoteLayerTree/RemoteScrollbarsController.mm:
(WebKit::RemoteScrollbarsController::scrollbarLayoutDirectionChanged):
(WebKit::RemoteScrollbarsController::mouseEnteredContentArea):
(WebKit::RemoteScrollbarsController::mouseExitedContentArea):
(WebKit::RemoteScrollbarsController::mouseMovedInContentArea):
(WebKit::RemoteScrollbarsController::shouldRegisterScrollbars const):
(WebKit::RemoteScrollbarsController::scrollbarWidthChanged):
(WebKit::RemoteScrollbarsController::updateScrollbarStyle):
(WebKit::RemoteScrollbarsController::scrollbarColorChanged):

Canonical link: https://commits.webkit.org/300377@main
